src/api/statements/LodexRunQuery.js

```diff
@@ -2,34 +2,48 @@ import { MongoClient } from 'mongodb';
 import config from 'config';
 import ezs from 'ezs';
 import set from 'lodash.set';
+
 import publishedDataset from '../models/publishedDataset';
 import field from '../models/field';
+import getPublishedDatasetFilter from '../models/getPublishedDatasetFilter';
 
 export const createFunction = MongoClientImpl =>
     async function LodexRunQuery(data, feed) {
         if (this.isLast()) {
             return feed.close();
         }
         const from = this.getParam('from', data.$from || 'dataset');
-        const query = this.getParam('query', data.$query || {});
-        const limit = this.getParam('limit', data.$limit || 10);
-        const skip = this.getParam('skip', data.$skip || 0);
-        const sort = this.getParam('sort', data.$sort || {});
+        const query = this.getParam('query', data.query || {});
+        const limit = this.getParam('limit', data.limit || 10);
+        const skip = this.getParam('skip', data.skip || 0);
+        const sort = this.getParam('sort', data.sort || {});
         const target = this.getParam('total');
 
         const handleDb = await MongoClientImpl.connect(
```

Comment:
Same as above, the connection should be initialized in a central place, and kept in the context/ in a singleton.